health/micro-ui/web/micro-ui-internals/packages/modules/health-hrms/src/components/pageComponents/CampaignAssignment.js

```
@@ -100,14 +103,38 @@ const CampaignsAssignment = ({ t, config, onSelect, formData }) => {
+                  setShowProjectPopup(true);
+                }}
+              />
+              {showProjectPopup && (
+                <PopUp
+                  className="project-search-popup"
+                  type="default"
+                  heading={t("HR_SELECT_PROJECT")}
+                  onOverlayClick={() => setShowProjectPopup(false)}
+                  children={[
+                    <div>
+                    <CardText style={{ margin: 0 }}>{t("ES_CAMPAIGN_UPDATE_TYPE_MODAL_TEXT") + " "}</CardText>
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)